blocks/cards/cards.css

```diff
@@ -149,3 +186,245 @@
   width: 38px;
   height:38px;
 }
+
+/* Styling for the 2up info card variant */
+
+  /* Styling for top level container for the 2up style of card (Flex container) */
+  .cards.info-card > ul {
+    display: flex;
+    flex-wrap: wrap;
+    padding-inline-start: 0;
+    grid-gap: 0;
+    margin-left: -20px;
+    margin-right: -20px;
+  }
+
+  /* Styling for top level container for each card (flex item) */
+  .cards.info-card > ul > li {
+    border: 0;
+    -webkit-box-flex: 0;
+    flex: 0 0 100%;
+    max-width: 100%;
+    box-sizing: border-box;
+  }
+
+  /* Add a margin to the first child since in narrow mode this is a single column */
+  .cards.info-card > ul > li:first-child {
+    margin-bottom: 5rem;
+  }
+
+  /* Div to contain the card elements */
+  .cards.info-card .card-container {
+    padding-left: 20px;
+    padding-right: 20px;
+    cursor: pointer;
+    display: -webkit-box;
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
```

Comment:
These -webkit prefixes are usually not needed anymore. please try if it works without.

Reply:
Sounds good, yeah think that was some cruft that I brought in while copying over the styles, I'll make a note to test without going forward.

Just tested pulling those props out and it did indeed make no difference to the rendering in browser, thanks!